streams/src/test/java/org/apache/kafka/streams/integration/RepartitionWithMergeOptimizingIntegrationTest.java

```diff
@@ -214,11 +213,11 @@ private int getCountOfRepartitionTopicsFound(final String topologyString) {
                                                               + "    Processor: KSTREAM-FILTER-0000000021 (stores: [])\n"
                                                               + "      --> KSTREAM-SINK-0000000020\n"
                                                               + "      <-- KSTREAM-MERGE-0000000004\n"
-                                                              + "    Sink: KSTREAM-SINK-0000000020 (topic: KSTREAM-MERGE-0000000004-optimized-repartition)\n"
+                                                              + "    Sink: KSTREAM-SINK-0000000020 (topic: KSTREAM-AGGREGATE-STATE-STORE-0000000005-repartition)\n"
```

Comment:
Same here and below needed to update the expected topology as now we use the topic name of the **_first_** merged repartition topic for the new optimized repartition topic.